components/dashboard/src/projects/NewProject.tsx

```diff
@@ -243,6 +245,8 @@ export default function NewProject() {
                 appInstallationId: String(repo.installationId),
             });
 
+            refreshProjects(project.teamId ? { teamId: project.teamId } : { userId: project.userId || "" });
+
             setProject(project);
```

Comment:
When adding a new project I noticed the re-sorting behavior.
I'm not :100:% sure, but could because of this line, while we do refresh async in the background (line above).
Should we simply remove this line? :thinking:

Reply:
Good catch! I'm not sure if it's that line (that state shouldn't be connected to the other page), but I'll take a look. I wonder if it's related to a new project not having a prebuild right away, but after a few seconds (and a triggered reload as you flipped between members/projects), we get the prebuild.

For this case, maybe we shouldn't be sorting the new project to the end (because it doesn't have a prebuild yet), but could use it's creation timestamp to sort by when no prebuild is present. I'll take a peek at it.